OpenRA.Mods.Common/Traits/AttackMove.cs

```diff
@@ -130,16 +141,34 @@ public override void Tick(World world)
 
 		public override string GetCursor(World world, CPos cell, int2 worldPixel, MouseInput mi)
 		{
-			var prefix = mi.Modifiers.HasModifier(Modifiers.Ctrl) ? "assaultmove" : "attackmove";
+			var prefix = mi.Modifiers.HasModifier(Modifiers.Ctrl);
```

Comment:
This variable no longer holds a prefix, so should be renamed to `isAssaultMove`.